libsolidity/interface/CompilerStack.cpp

```diff
@@ -386,7 +389,7 @@ bool CompilerStack::analyze()
 
 	if (noErrors)
 	{
-		m_stackState = AnalysisSuccessful;
```

Comment:
Why rename this to `AnalysisPerformed` if it is only set in the success case?

Reply:
> Why rename this to AnalysisPerformed if it is only set in the success case?

The resname was because that is what you suggested. I had realized that the the suggestion also implied going over the code and modifying the behavior.  Yes, that is my mistake but you would help me out by making this more explicit. Often your comments tend to be of the form that I need to infer what you fully mean.

Reply:
Comments by reviewers are never meant as "do it exactly like that". Reviewers can only make suggestions. The code author always knows the extent of their changes best. Reviewers notice rough edges in the code. Sometimes, the author just overlooked something, but often there are situations where these rough edges cannot be avoided easily. In these cases, authors and reviewers have to find a good solution together.